src/FlashMessage/FlashMessage.tsx

```diff
@@ -0,0 +1,160 @@
+    };
+
+    const removeFadeout = () => {
+        setFadeoutTimer('unset');
+        if (timeoutHandler.current) {
+            clearTimeout(timeoutHandler.current);
+        }
+    };
+
+    const initialRender = () => divRef.current == null;
+
+    return (
+        <div
+            ref={divRef}
+            style={{
+                backgroundColor: getBackgroundColorFromVariant(variant),
+                color: colors.white,
+                zIndex: 1000,
+                animation: initialRender() ? 'slide-in 1s' : 'unset',
+                width: '100%',
+                padding: '16px',
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+            }}
+            className={`flash-variant-${variant}`}
+            onMouseEnter={removeFadeout}
+            onMouseLeave={addFadeout}
+        >
+            <div
+                style={{
+                    width: '100%',
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                }}
+            >
+                {message}
+                <div onClick={close}>
+                    <Icon path={mdiClose} size={0.8} />
+                </div>
+            </div>
+            {dismissTime != null ? (
+                <div
+                    style={{
+                        backgroundColor: colors.white,
+                        width: 'calc(100% + 32px)',
+                        height: '2px',
+                        margin: '-16px',
+                        marginTop: '8px',
+                        animation:
+                            fadeoutTimer !== 'unset'
+                                ? `flash-message ${dismissTime}ms linear`
+                                : 'unset',
+                    }}
+                />
+            ) : null}
+        </div>
+    );
+};
+
+const FlashMessages = () => {
+    const messages = useSelector(getMessages);
+    const showMessages = messages?.length > 0;
+
+    if (!showMessages) return null;
+
+    return (
+        <div
+            style={{
+                width: '256px',
+                gap: '16px',
+                marginBottom: '32px',
+                display: 'flex',
+                flexDirection: 'column-reverse',
+            }}
+            className="message-container"
+        >
+            {messages.map(flashMessage => (
+                <FlashMessage
+                    key={flashMessage.id}
+                    flashMessage={flashMessage}
+                />
+            ))}
+        </div>
+    );
+};
+
+const getBackgroundColorFromVariant = (variant: FlashMessageVariant) => {
+    switch (variant) {
+        case 'error':
+            return colors.red;
+        case 'success':
+            return colors.green;
+        case 'info':
+            return colors.nordicBlue;
+        case 'warning':
+            return colors.orange;
+
+        default:
+            return '' as never;
```

Comment:
Consider using `'unset'` as a fallback css value

Reply:
I agree, but I would like it to return `as never`, to indicate to the programmer that we only accept the explicitly handled variants.